service/routes.py

```diff
@@ -186,6 +186,7 @@ def put(self, promotion_id):
             abort(status.HTTP_404_NOT_FOUND, 'Promotion with id [{}] was not found.'.format(promotion_id))
 
         data = api.payload
+        print(data)
```

Comment:
Not sure if this print statement is required

Reply:
Right. I forgot to remove that from when I was debugging. Removed the unnecessary print statement from routes.py